CHANGELOG.md

```diff
@@ -3,10 +3,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [2.4.6] - 2024-07-12
 
 - Update dependencies in [#200](https://github.com/cybozu-go/placemat/pull/200)
 
+## [Unreleased]
+
```

Comment:
The order of the headings is wrong.
And `[2.4.6]` is a dead link.

Reply:
Sorry for that.
I fixed it now.